includes/REST_API/Templates_Controller.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Class Templates_Controller
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\REST_API;
+
+/**
+ * Templates_Controller class.
+ *
+ * Will handle template-specific logic in the future.
+ */
+class Templates_Controller extends Stories_Base_Controller {
```

Comment:
Can we just use `Stories_Base_Controller` when we define the post type and get rid of this file?

Reply:
sure, no strong opinion on this. will change.